lib/zclient.c

```diff
@@ -2593,6 +2739,175 @@ int lm_label_manager_connect(struct zclient *zclient, int async)
 	return -1;
 }
 
+/**
+ * Connect to srv6 manager in a syncronous way
```

Comment:
always suspicious of 'synchronous' - I don't see this being used: what is the use-case, and why does it have to be synchronous/blocking ?

Reply:
Thank you for your comment.

I think your suspicion is correct.
The srv6_manager_connect function was a function I added on purpose by "referring to label manager etc." which we have been discussing for a while now.

So this function is not needed at the moment.

I deleted following code and its depending lines. at [this commit](https://github.com/FRRouting/frr/pull/5865/commits/4c94e2801dd5fae1af264d3392d21482cc14a329).

* function srv6_manager_connect
* function zread_srv6_manager_connect
* function zsend_srv6_manager_connect_response
* enum ZEBRA_SRV6_MANAGER_CONNECT

> I guess I prefer (1): do the right thing in new code, even if there is old code that does something wrong. just as you were tricked by the old code, someone else will be tricked in the future if we don't stop doing the wrong thing?

As you advised above in another comment, this srv6_manager_connect is not needed as well.

Reply:
If @mjstapp does not have any other comments about deprecating this srv6_manager_connect, please click the "resolve conversation" button.

Reply:
I fixed this lines so I make this conversation as resolved.